detox/package.json

```diff
@@ -42,6 +42,7 @@
     "commander": "^2.15.1",
     "fs-extra": "^4.0.2",
     "get-port": "^2.1.0",
+    "image-size": "^0.6.3",
```

Comment:
As discussed in the comments, this should come from native and this is unnecessary.